data-pipeline/handlers/process_file.rb

```diff
@@ -21,6 +21,9 @@ def process(key:, bucket:)
         if next_bucket == @environment['AMR_DATA_BUCKET']
           file_body = StringIO.new
           file.body.each_line do |line|
+            next if line.strip.empty?
```

Comment:
This gets updated in a later PR as we should remove the UTF8 nulls first and also remove any invalid characters...